src/lib/runtime/mounts/dev/dev.c

```diff
@@ -105,6 +105,20 @@ int _singularity_runtime_mount_dev(void) {
         bind_dev(sessiondir, "/dev/random");
         bind_dev(sessiondir, "/dev/urandom");
 
+        /* If /dev/infiniband exists include it */
+        if ( is_dir("/dev/infiniband") == 0 ) {
+            int ret;
```

Comment:
Doesn't look like this is being used anywhere.  Copy and paste mistake?

Reply:
No ... I _was_ going to check the return value and check that in an `if` ... but then changed how I was checking and forgot to remove that.